gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -76,16 +79,15 @@ def get_ranges(self):
             --> Add range service proxy using thruster names
                 --> This is not necessary, since they are all the same thruster
         '''
-        range_service = 'thrusters/thruster_range'
-        rospy.logwarn("Waiting for service {}".format(range_service))
-        rospy.wait_for_service(range_service)
-        rospy.logwarn("Got {}".format(range_service))
+        thruster_info_service = 'thrusters/thruster_info'
```

Comment:
This works, but seems unnecessary when all of this information is in the thruster ros param it already has and saves the service calls.